roles/ceph-config/templates/ceph.conf.j2

```diff
@@ -82,18 +82,14 @@ filestore xattr use omap = true
 {% if osd_objectstore == 'bluestore' %}
 {% set _num_osds = num_osds | default(0) | int %}
 [osd]
-{% if is_hci | bool and _num_osds > 0 %}
+{% if is_hci and _num_osds > 0 and not osd_memory_target is defined %}
```

Comment:
You have to put back the `bool` filter there

Reply:
Right, will fix this.